.github/workflows/gtn-news.yml

```diff
@@ -0,0 +1,166 @@
+      # `peter-evans/create-pull-request@v5` does not create duplicate PRs by default
+      # (see https://github.com/peter-evans/create-pull-request#action-behaviour)
+      - name: Get files touched by existing GitHub PRs that are newer than the oldest article in the feed.
+        id: files
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          files="$(gh pr list --repo https://github.com/${{ github.repository }} -s all -S "created:>=${{ steps.parse.outputs.date }}" --json files)"
+          echo files="$files" > $GITHUB_OUTPUT
+      - name: Filter out GTN news for which a PR may already exist.
+        id: filter
+        shell: python -u {0}
+        run: |
+          import json
+          import os
+
+          files=json.loads("""
+              ${{ steps.files.outputs.files }}
+          """)
+          files={file["path"] for item in files for file in item["files"]}
+
+          posts=json.loads("""
+              ${{ steps.parse.outputs.posts }}
+          """)
+
+          existing = {
+              name
+              for name in posts
+              if any(name in file for file in files)
+          }
+          for name in existing:
+              del posts[name]
+
+          with open(os.environ["GITHUB_OUTPUT"], 'w') as file:
+              print("posts=%s" % json.dumps(posts), file=file)
+
+      - name: Prepare matrix for `pull_requests` job.
+        id: matrix
+        shell: python -u {0}
+        run: |
+          import json
+          import os
+
+          posts=json.loads("""
+              ${{ steps.filter.outputs.posts }}
+          """)
+
+          with open(os.environ["GITHUB_OUTPUT"], 'w') as file:
+              print(
+                  "matrix=%s" % f"{json.dumps({'name': list(posts)})}",
+                  file=file
+              )
+    outputs:
+      posts: ${{ steps.filter.outputs.posts }}
+      matrix: ${{ steps.matrix.outputs.matrix }}
+
+  pull_requests:
+    name: Create pull request
+    runs-on: ubuntu-latest
+    needs: collect
+    strategy:
+      fail-fast: false
+      matrix: ${{fromJson(needs.collect.outputs.matrix)}}
+    steps:
+      - name: Check out the Galaxy Hub.
+        uses: actions/checkout@v3
+        with:
+          ref: master
+
+      - name: Create a Galaxy Hub post.
+        id: post
+        shell: python -u {0}
+        run: |
+          import json
+          import os
+          import textwrap
+
+          post = json.loads("""
+              ${{ needs.collect.outputs.posts }}
+          """)["${{ matrix.name }}"]
+          post = textwrap.dedent(f"""
+              ---
+              site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+              date: "{post['published']}"
+              tags: [training, gtn-news]
+              title: "{post['title']}"
+              supporters:
+              - denbi
+              - elixir
+              - gallantries
+              external: '{post["link"]}'
+              ---
```

Comment:
This needs to be double-checked.

Reply:
yeah not sure if we want to do something better with `supporters`, the GTN has a more extensive+accurate list, but keeping that in sync would be annoying and maybe not worth it.

Also the site selection, right now only EU ones but since it's now on the hub maybe it makes sense to distribute to all galaxies? (Or we can set a flag on the GTN side somehow to indicate that it's applicable for everyone?)

Reply:
I better make this no longer a draft and request reviews from several people so that they also contribute to this discussion.

Reply:
Not sure the supporters will even be shown on the Hub with external links. I would either remove them completely or add something like "all" which we do not support at the moment.

`site:` I would definitely go for `all`.